draft-ietf-tls-esni.md

```diff
@@ -717,61 +735,62 @@ then present a certificate based on the public name, without echoing the
 Depending on whether the client is configured to accept the proxy's certificate
 as authoritative for the public name, this may trigger the retry logic described
 in {{handle-server-response}} or result in a connection failure. A proxy which
-is not authoritative for the public name cannot forge a signal to disable ESNI.
+is not authoritative for the public name cannot forge a signal to disable ECHO.
 
-A non-conformant MITM proxy which instead forwards the ESNI extension,
+A non-conformant MITM proxy which instead forwards the ECHO extension,
 substituting its own KeyShare value, will result in
 the client-facing server recognizing the key, but failing to decrypt
 the SNI. This causes a hard failure. Clients SHOULD NOT attempt to repair the
 connection in this case.
 
+# TLS and HPKE Ciphersuite Mapping {#hpke-map}
+
+Per {{RFC8446}, TLS ciphersuites define an AEAD and hash algorithm. In contrast,
+HPKE composes AEAD algorithms and key derivation functions. The table below lists
+the mapping between ciphersuites and HPKE identifiers.
+
+| TLS Ciphersuite | HPKE AEAD | HPKE KDF |
+|:-------|:------------------------|:----------|
+| TLS_AES_128_GCM_SHA256 | AES-GCM-128 | HKDF-SHA256 |
+| TLS_AES_256_GCM_SHA384 | AES-GCM-256 | HKDF-SHA256 |
+| TLS_CHACHA20_POLY1305_SHA256 | ChaCha20Poly1305 | HKDF-SHA256 |
+| TLS_AES_128_CCM_SHA256 | AES-CCM-128 | HKDF-SHA256 |
+| TLS_AES_128_CCM_8_SHA256 | AES-CCM-128-8 | HKDF-SHA256 |
+
```

Comment:
You're going to want to signal this upfront.

Reply:
Do you mean reference this sooner, or elaborate on the mapping?